lib/utils/ICETelescope.cpp

```diff
@@ -129,8 +129,7 @@ freq_id_t ICETelescope::to_freq_id(stream_t stream, uint32_t ind) const {
         case 8: // 16 ICEBoards (256 elements) e.g. Pathfinder/HIRAX-128
```

Comment:
This also applies to GBO and Hat Creek, right?

Reply:
Yes, I'll add those just for completeness.